src/pymatgen/electronic_structure/plotter.py

```diff
@@ -371,7 +371,7 @@ def _make_ticks(self, ax: plt.Axes) -> plt.Axes:
     def _get_branch_steps(branches):
         """Find discontinuous branches."""
         steps = [0]
-        for b1, b2 in zip(branches[:-1], branches[1:], strict=False):
```

Comment:
I think the slicing in `branches[:-1]` is doing nothing here, but pin it in case I misunderstand anything.

To confirm:
```python
from itertools import pairwise


branches = ["a", "b", "c", "d"]

print("With zip:")
for b1, b2 in zip(branches[:-1], branches[1:], strict=False):
    print(f"{b1} + {b2}")

print("With itertool:")
for b1, b2 in pairwise(branches):
    print(f"{b1} + {b2}")
```

Reply:
yes, `branches[:-1]` here was redundant as `zip` would have stopped before the last element anyway

Reply:
Thanks for confirming!